### ISSUE TITLE
Use of HTML in description strings and general annotations

### ISSUE THREAD

**Issue by henrikt-ma** [modelica/ModelicaSpecification#3051]
Some time ago it was clarified how HTML content can be included in the `Documentation` annotation (see #2593), but it was missed that the same kind of formulations that were improved then also existed in the _Modelica Concrete Syntax_ chapter:
https://github.com/modelica/ModelicaSpecification/blob/35fb5f276a6bd993496d6073b10e3426ff762905/chapters/syntax.tex#L64

This was reported in feedback to https://github.com/modelica/ModelicaSpecification/pull/3045#discussion_r755270942, but it was agreed that it shouldn't be resolved as part of that PR.

Thus, this issue seeks clarification of these questions:
- Are there any other annotations than `Documentation` where HTML tags are supported?  In exactly which forms should they be recognized in the other annotations?
- Exactly how can a `description-string` make use of HTML?

**Comment by beutlich** [comment]
> * Are there any other annotations than `Documentation` where HTML tags are supported?

I touched that question for the `label` annotation in #2915.

**Comment by HansOlsson** [comment]
One obvious use of html in description string is something like:
`parameter Real beta2 "<html>The value for &beta;<sup>2</sup></html>";`

**Comment by beutlich** [comment]
> One obvious use of html in description string is something like:
> `parameter Real beta2 "<html>The value for &beta;<sup>2</sup></html>";`

Which already can be expressed by current Modelica:
`parameter Real beta2 "The value for β²";`

**Comment by henrikt-ma** [comment]
> One obvious use of html in description string is something like: `parameter Real beta2 "<html>The value for &beta;<sup>2</sup></html>";`

Sure.  For the description strings we just need the same clear rules for how to use the HTML tags as we developed for the `Documentation` annotation in [18.2.1](https://specification.modelica.org/master/annotations.html#annotation-info-revisions).

**Comment by henrikt-ma** [comment]
> > * Are there any other annotations than `Documentation` where HTML tags are supported?
> 
> I touched that question for the `label` annotation in #2915.

Right, so here we have a potential future annotation where HTML could be useful.

What makes me wonder about having a rule about support for HTML in annotations in general is how widely applicative it is – I wonder if there's any tool today that supports all of these:
- 18.6.5.5: `Text`
- 18.7: `missingInnerMessage`, `obsolete`, `unassignedMessage`
- 18.9.2: `Authorization.licensor`, `License.licensee`
- 7.3.4: `choices.choice`

I probably missed some of the annotations where HTML could be meaningful, but one can also argue whether some of the annotations above would really be meaningful places for HTML content.  The unclarity of this situation is what makes me think that it would be better to remove the generic rule about HTML being supported in annotations, and replace it with specific remarks about HTML support for selected annotations where we really want it.

Finally, it would be nice if we could refactor the presentation so that as much as possible about the strings with HTML content is described in one place, and that all places where HTML should be supported then refer to the common description.

**Comment by HansOlsson** [comment]
Hans: Make a PR for exact same semantics for description-strings (must start with <html> etc).
